ci/jenkins/test.sh

```diff
@@ -429,6 +444,131 @@ function deliver_antrea_windows {
+    e2e_images=("k8sprow.azurecr.io/kubernetes-e2e-test-images/agnhost:2.40" "k8sprow.azurecr.io/kubernetes-e2e-test-images/jessie-dnsutils:1.5" "k8sprow.azurecr.io/kubernetes-e2e-test-images/nginx:1.14-2")
+
+    for i in "${!harbor_images[@]}"; do
+        ctr -n=k8s.io images delete "${antrea_images[i]}"
+        ctr -n=k8s.io images pull "${DOCKER_REGISTRY}/antrea/${harbor_images[i]}"
+        ctr -n=k8s.io images tag "${DOCKER_REGISTRY}/antrea/${harbor_images[i]}" "${antrea_images[i]}"
+    done
+    echo "===== Deliver Antrea to Linux worker nodes and pull necessary images on worker nodes ====="
+    kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 !~ role && $1 !~ /win/ {print $6}' | while read IP; do
+        rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" antrea-ubuntu.tar jenkins@${IP}:${WORKDIR}/antrea-ubuntu.tar
+        ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "${CLEAN_STALE_IMAGES_CONTAINERD}; ctr -n=k8s.io images import ${WORKDIR}/antrea-ubuntu.tar" || true
+
+        for i in "${!harbor_images[@]}"; do
+            ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "ctr -n=k8s.io images pull ${DOCKER_REGISTRY}/antrea/${harbor_images[i]} && ctr -n=k8s.io images tag ${DOCKER_REGISTRY}/antrea/${harbor_images[i]} ${antrea_images[i]}" || true
+        done
+        # Pull necessary images in advance to avoid transient error
+        for image in "${common_images[@]}"; do
+            ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "ctr -n=k8s.io images pull ${image}" || true
+        done
+        # Pull necessary images for Kubernetes conformance test
+        for i in "${!k8s_images[@]}"; do
+            ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "ctr -n=k8s.io images pull ${k8s_images[i]} && ctr -n=k8s.io images tag ${k8s_images[i]} ${e2e_images[i]}" || true
+        done
+    done
+
+    echo "===== Build Antrea Windows on Windows Jumper Node ====="
+    echo "==== Reverting Windows VM ${WIN_JUMPER} ====="
+    govc snapshot.revert -vm ${WIN_JUMPER} win-initial
+    winVMIP=""
+    for i in `seq 10`; do
+        winVMIP=$(govc vm.ip -wait=2m -a ${WIN_JUMPER})
+        if [[ $winVMIP != "" ]]; then
+            echo "Windows VM ${WIN_JUMPER} powered on"
+            break
+        fi
+        echo "Windows VM ${WIN_JUMPER} failed to power on"
+        govc vm.power -on ${WIN_JUMPER} || true
+    done
+    if [[ $winVMIP == "" ]]; then
+        echo "Windows VM ${WIN_JUMPER} didn't power on after 10 tries, exiting"
+        exit 1
+    fi
+    winVMIP=${winVMIP#*,}
+    # Windows VM is reverted to an old snapshot so computer date needs updating.
+    for i in `seq 24`; do
+        sleep 5
+        ssh -o StrictHostKeyChecking=no -n Administrator@${winVMIP} "W32tm /resync /force" | grep successfully && break
+    done
```

Comment:
I think it can call `revert_snapshot_windows ${WIN_JUMPER}` too

Reply:
Updated.